parquet-pig/src/test/java/org/apache/parquet/pig/TestDecimalUtils.java

```diff
@@ -60,12 +60,12 @@ public void testBinaryToDecimal() throws Exception {
     // Test LONG
     testDecimalConversion(Long.MAX_VALUE, 19, 0, "9223372036854775807");
     testDecimalConversion(Long.MIN_VALUE, 19, 0, "-9223372036854775808");
-    testDecimalConversion(0L, 0, 0, "0.0");
+    testDecimalConversion(0L, 0, 0, "0");
```

Comment:
Why do these two lines need change?

Reply:
For this use case, 0 is the correct value. Using double to construct BigDecimal previously may lead to potential errors.